jenkins/create_state_county_country_basemap

```diff
@@ -0,0 +1,76 @@
+pipeline {
+  agent {
+        node {
+            label 'node:slave'
+        }
+    }
+  parameters {
+        gitParameter name: 'BRANCH_TAG',
+                     type: 'PT_BRANCH_TAG',
+                     defaultValue: 'master'
+        choice(choices: ['test', 'qa', 'prod'], description: 'Tier to deploy tiles to', name: 'TIER')
```

Comment:
Could drop the tier parameter if you're not using it, although maybe it would be useful to be able to deploy to the test tier as well?  Up to you

Reply:
yeah, I considered that but it doesn't seem like the viz itself uses any tiles from the test tier with the exception of the hru tiles